### DISCUSSION TITLE
How to handle redirect for update.UserAttributes

### DISCUSSION THREAD

**Discussion by theShreyans** [supabase/supabase#9662] [Questions]
Currently when I try to update email using:
```dart
 await Supabase.instance.client.auth
              .update(UserAttributes(email: newEmail));
```
It sends a link to the new email for confirmation but when I click on it, it doesn't redirect to the app instead I get the message no api key found in the request. However, the email does get updated.

I was trying to figure out how to redirect users back to the flutter app. In other functions there is an option to URL redirect but not with update.UserAttributes.

**Comment by mansueli** [comment]
Hi there, 

This is being discussed here: https://github.com/supabase/gotrue-js/pull/410

I will be closing this one, but you can go there and comment /discuss it further.
